documentation/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM php:5.6-apache
+
+# Allow rewrites in Apache httpd
+RUN a2enmod rewrite
+
+# Get some packages, especially Git, but also some convenience items
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        curl \
+        git \
+        less \
+        vim \
+    && rm -rf /var/lib/apt/lists/*
+
+# Add Kirby base
+RUN mkdir -p /var/www/html/ \
+    && git clone --recursive https://github.com/getkirby/starterkit.git /var/www/html \
```

Comment:
It might be reasonable to clone something that _isn't_ the Kirby Starter Kit, given the number of things I do below to get a basic theme without extras.